### PR TITLE
fix: tabs_animation_bug

### LINKED ISSUE #23050: Tabs - In Safari, the tab content which is not the first disappears
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://codepen.io/sevenumr/pen/NWGKMwr?editable=true&editors=001](https://codepen.io/sevenumr/pen/NWGKMwr?editable=true&editors=001)

### Steps to reproduce
Switch to the tab which is not the first

### What is expected?
The Tab Content shows correctly

### What is actually happening?
The Tab Content flashed and then disappear

| Environment | Info |
|---|---|
| antd | 4.1.1 |
| React | 16 |
| System | macOS Mojave 10.14.1 |
| Browser | Safari 13.1 |

### PR DESCRIPTION
**Pull request by imhxc**
### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
fix [#23050](https://github.com/ant-design/ant-design/issues/23050)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Fix the issue that Tabs component cannot be displayed in Safair browser.  |
| 🇨🇳 Chinese |    修复 Tabs 开启动画的情况下，在 Safair 浏览器下的无法显示的问题。     |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
image

**Comment by afc163** [comment]
Rebase master to fix ci.

**Comment by imhxc** [comment]
@afc163 
I modified the place you just mentioned, but still some checks were not successful.
I don't know if these errors are related to me, I hope to get help.

**Comment by afc163** [comment]
Rebase master to fix ci.

File: components/tabs/style/index.less [hunk 1]
```diff
@@ -261,10 +261,10 @@
   .@{tab-prefix-cls}-top-content,
   .@{tab-prefix-cls}-bottom-content {
     width: 100%;
-
     > .@{tab-prefix-cls}-tabpane {
       flex-shrink: 0;
       width: 100%;
+      -webkit-backface-visibility: hidden;
```
Comment (yoyo837) [inline components/tabs/style/index.less:267]: 是不是需要再补个标准属性？`backface-visibility`

Comment (afc163) [reply components/tabs/style/index.less:267]: 不需要了，特别给 safari 老版本做兼容用的。

**Comment by nick-ChenZe** [comment]
@afc163 this bug still in 3.26.16, this pr should cherry-pick to 3.x.
thanks

**Pull request merged**